### PR TITLE
Fixes #3900 - add file size to Add-on metadata

### LINKED ISSUE #3900: Expose the Addon size in the Add-on metadata
Refiled from https://github.com/mozilla/addons/issues/535

On the old site we exposed the add-on download size. 

ublock_origin____add-ons_for_firefox

However, in the new frontend this detail is missing. It's useful for knowing the size of the download ahead of time so it would be good to re-instate this data.

### PR DESCRIPTION
**Pull request by rohitsanj**
Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.

Please delete anything that isn't relevant to your patch.

- [x] This PR relates to an existing open issue and there are no existing PRs open for the same issue.
- [x] Add `Fixes #ISSUENUM` at the top of your PR.
- [x] Add a description of the changes introduced in this PR.
- [x] The change has been successfully run locally.
- [ ] Add tests to cover the changes added in this PR.
- [x] Add before and after screenshots (Only for changes that impact the UI).

Description:
@gjaya79 and I added the file size to meta data of the Add-on

Fixes [#3900](https://github.com/mozilla/addons-frontend/issues/3900)
### Before
before

### After
after

### CODE REVIEW

File: src/core/reducers/addons.js [hunk 1]
```diff
@@ -242,13 +241,21 @@ export function createInternalAddon(
...
     currentVersion.files.length > 0
   ) {
+    const { files } = currentVersion;
+    const fileSizeInBytes = files
+      .map((file) => file.size)
+      .reduce((accumulator, size) => accumulator + size);
+    const fileSize =
+      fileSizeInBytes <= 10 ** 5
+        ? `${(fileSizeInBytes / 10 ** 3).toFixed(1)} KiB`
+        : `${(fileSizeInBytes / 10 ** 6).toFixed(1)} MiB`;
```
Comment (willdurand) [inline src/core/reducers/addons.js]: We have a `formatFilesize()` helper in `src/core/i18n.utils.js` to translate a file size in bytes into a localized user-friendly format.

In addition, I think we should avoid adding two many props on the `addons` object. We could probably compute the size of the version in `AddonsMoreInfo`?

**Comment by willdurand** [comment]
@rohitsanj thanks for your first patch!

File: src/core/reducers/addons.js (hunk 1 above)
Comment (rohitsanj) [reply src/core/reducers/addons.js]: Thanks for reviewing. 
Sure, I'll send in another patch with the aforementioned changes soon.

**Comment by caitmuenster** [comment]
Hey @rohitsanj! It looks like you are really close to finishing this one off. Want to get it over the finish line?

**Comment by rohitsanj** [comment]
Hey @caitmuenster 
I was busy with something else, but I managed to refactor the code and get it working. I could not find time to write a test case.

**Comment by muffinresearch** [comment]
@rohitsanj I'm going to close this out since time to add a test case is going to be difficult as per your previous comment. Please re-open if you are able to pick up where you left off.

**Pull request closed without merging**